README.md

```diff
@@ -25,9 +26,55 @@ import express from 'express';
 
 app = express();
 
-app.use(error_handler(logger));
+app.use(error_handler(logger, mailer, {
+    unique: true,                   // optional defaults to true
+    to: 'dev@mail.com',             // required if mailer is present
+    from: 'app@mail.com',           // required if mailer is present
+    subject: 'MyApp > Server Error' // required if mailer is present
+    pretend: false                  // optional, set to true to pretending sending of emails
+}));
 ```
 
+`mailer` - should be an instance of [anytv-mailer](https://github.com/anytv/anytv-mailer)
+
+
+# Setting up the mailer
+1. Make sure mailer is already configured
+2. Create a template for the error, `${ TEMPLATES_DIR }/error/html.ejs`
+3. `err` object will be passed onto the template you can render it using `<%- JSON.stringify(err, null, '\t') ->`
```

Comment:
Let's have a default template provided?

I guess there's only one template that can be used any given time?